internal/cmd/egctl/testdata/translate/out/default-resources.all.yaml

```diff
@@ -17,6 +17,11 @@ envoyProxyForGatewayClass:
             socket_address:
               address: 127.0.0.1
               port_value: 19000
+        cluster_manager:
+          local_cluster_name: local_cluster
+        node:
+          locality:
+            zone: $(ENVOY_SERVICE_ZONE)
```

Comment:
This gets replaced with the container's env var or renders as an empty string leading to an empty locality. Still valid configuration and would be the same end result as today.